frontend/src/app/PrimeErrorBoundary.js

```diff
@@ -18,10 +19,7 @@ export default class PrimeErrorBoundary extends React.Component {
 
   render() {
     if (this.state.hasError) {
-      const { onError } = this.props;
-      return typeof onError === "function"
-        ? onError(this.state.error)
-        : React.createElement(onError);
+      return React.createElement(ErrorPage);
```

Comment:
Nice, no need to pass it as a prop :+1:

Reply:
I wrote it this way to get around writing specific error tests for App and PatientApp, but let's call it a feature not a bug 😂